packages/algorand-js/src/interpreter/opcode.ts

```diff
@@ -71,4 +76,33 @@ export class Op {
 
     return byteString.slice(Number(start), Number(end));
   }
+
+  assertAccountDefined (a?: AccountState): AccountState {
+    if (a === undefined) {
+      throw new TealError(ERRORS.TEAL.ACCOUNT_DOES_NOT_EXIST);
+    }
+    return a;
+  }
+
+  assertAppDefined (appId: number, interpreter: Interpreter): SSCParams {
+    const globalDelta = interpreter.globalApps.get(appId);
+    if (globalDelta === undefined) {
+      throw new TealError(ERRORS.TEAL.APP_NOT_FOUND);
+    }
+    return globalDelta;
+  }
```

Comment:
this should be a method of a runtime. You can do this later, once you will create runtime -> in that case please add TODO in the file, and add a subtask to the Runtime PoC pivotal task.

Reply:
ok